client/src/modules/inventory/list/modals/log.modal.html

```diff
@@ -0,0 +1,40 @@
+<div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate> FORM.LABELS.CHANGES</li>
+      <li class="title text-capitalize">
+         
+          {{ModalCtrl.inventory.label}}
+      </li>
+    </ol>
+  </div>
+  <div class="modal-body">
+    <table class="table table-bordered">
+      <thead>
+        <tr>
+          <th translate>FORM.LABELS.DATE</th>
+          <th translate>FORM.LABELS.CODE</th>
+          <th translate>FORM.LABELS.NAME</th>
+          <th translate>FORM.LABELS.PRICE</th>
+          <th translate>INVENTORY.LIST_GROUP</th>
+          <th translate>INVENTORY.LIST_TYPE</th>
+          <th translate>FORM.LABELS.USER</th>
+        </tr>
+        <tr ng-repeat="log in ModalCtrl.logs"
+            ng-class="{'text-primary': log.text.action=='CREATION'}">
+          <td>{{log.log_timestamp | date : "dd/MM/yyyy HH:mm:ss" }}</td>
+          <td>{{log.text.code}}</td>
+          <td>{{log.text.text}}</td>
+          <td>{{log.text.price}}</td>
+          <td>{{log.text.inventoryGroup.name || ''}}</td>
+          <td>{{log.text.inventoryType.text || ''}}</td>
+          <td>{{log.userName}}</td>
+        </tr>
+      </thead>
+    </table>
+  </div>
```

Comment:
Why you do not follow this style :  

![image](https://user-images.githubusercontent.com/5445251/58087806-38958500-7bb9-11e9-92b2-38930758d538.png)

Reply:
Could we have these information : 

- [ ] What change changed?
- [x] Who changed it?
- [x] When did it change?
- [ ] What value did it have before?

![image](https://user-images.githubusercontent.com/5445251/58088777-48ae6400-7bbb-11e9-98a5-4325399c765e.png)

- Is it possible to track change of all inventory attributes not only : code, name, price, type, group, user, date ; it means include also the track of : consumable, locked, default quantity, unit, etc.
- Is it possible to show a meaningful message when there is no change instead of having nothing
- Is it possible to have for the attribute which changed, its previous value such as in the issue description